src/ui/popup/theme/controls/index.tsx

```diff
@@ -9,6 +9,7 @@ import Scheme from './scheme';
 import Scrollbar from './scrollbar';
 import SelectionColorEditor from './selection';
 import Sepia from './sepia';
+import StyleStandardElements from './style-standard-elements';
```

Comment:
Let's make a file name and a component function name to correspond with the setting name.